next/components/forms/TextAreaField.tsx

```diff
@@ -26,18 +26,23 @@ const TextAreaField = ({
   description,
   tooltip,
   required,
-  value = '',
+  value,
   disabled,
   className,
   ...rest
 }: TextAreaBase) => {
-  const [valueState, setValueState] = useState<string>(value)
+  const [valueState, setValueState] = useState<string>('')
   const ref = React.useRef<HTMLTextAreaElement>(null)
+
+  const getValue = () => {
+    return rest.onChange ? value : valueState
+  }
```

Comment:
I don't see the reason for putting this into function (?), instead you can do something like

```
const displayValue = rest.onChange ? value : valueState
```

and then use `displayValue`